cypress/integration/login.js

```diff
@@ -1,94 +1,115 @@
+import LoginPage from '../objects/loginPage';
+const loginPage = new LoginPage();
+
+import TaskPage from '../objects/taskPage';
+const taskPage = new TaskPage();
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
         
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+        loginPage.visit();
+        loginPage.getUserValidationIndicator().should('not.visible')
+        loginPage.getSubmitButton().click();
+        loginPage.getUserValidationIndicator().should('be.visible')
     })
 
     it('Check login as Admin and if the day is today', function () {
 
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Edvinas Kveselys"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="Admin"]').click();
-        cy.get('[type="submit"]').click();
+        loginPage.visit();
+        loginPage.getLoginUserId().click({force:true});
+        loginPage.getSpecificUserFromDropDown("Edvinas Kveselys").click();
+        loginPage.getLoginRole().click({force:true});
+        loginPage.getSpecificRoleFromDropDown("Admin").click();
+        loginPage.getSubmitButton().click();
 
-        cy.url().should('include', '/time-logging');
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Edvinas Kveselys');
-        cy.get('.main-nav').find('li').should('have.length', 6);
+        loginPage.urlShouldInclude("/time-logging");

```

Comment:
Methods with time logging elements should be put in `timeLoggingPage.js`. Also elements that are shared between pages (like navigation bar) should be put in separate file like `common.js` or `pageComponents.js`